src/openvino.cc

```diff
@@ -632,12 +639,170 @@ ModelState::ValidateOutputs()
 TRITONSERVER_Error*
 ModelState::AutoCompleteConfig()
 {
-  // TODO: Add the support for auto completing config for this backend.
-  LOG_MESSAGE(
-      TRITONSERVER_LOG_WARN,
-      (std::string("skipping model configuration auto-complete for '") +
-       Name() + "': not supported for openvino backend")
-          .c_str());
+  // Read OV model for autocomplete
+  std::string artifact_name;
+  RETURN_IF_ERROR(
+      ModelConfig().MemberAsString("default_model_filename", &artifact_name));
+  std::string model_path;
+  THROW_IF_BACKEND_INSTANCE_ERROR(ReadModel(artifact_name, &model_path));
+  model_read_ = false;  // Re-read model after autocomplete
+
+  // Get OV model inputs and outputs
+  std::vector<ov::Output<ov::Node>> model_inputs;
```

Comment:
This is so strange. Why is the input node of type `ov::Output<ov::Node>` ?
Shouldn't it be ov::Input or something?

Reply:
Unfortunately, this is just how OpenVINO designed their interface for getting model's input and output.
Documentations: 
[`std::vector<ov::Output<ov::Node>> inputs()`](https://docs.openvino.ai/2022.1/classov_1_1Model.html#doxid-classov-1-1-model-1ac28a4c66071e165c4f98906ab489e5d5)
[`std::vector<ov::Output<ov::Node>> outputs()`](https://docs.openvino.ai/2022.1/classov_1_1Model.html#doxid-classov-1-1-model-1af6e381008712ce22d6f4b93b87303dd8)